README.md

```diff
@@ -23,11 +35,18 @@ The goal of the project was to study and practice the three types of programming
 ## Misc
 
 - The project is almost a identical replica - the only thing that wasn't implemented was the enemies, due to the project's deadline.
-
+- Yes, we plan on making updates in the future.
 
 # ![New Relic logo](https://newrelic.com/static-assets/images/icons/avatar-newrelic.png) New Relic DevOpsDays Challenge
 
 Win in 3 simple steps:
 1. Clone this repository, and complete the New Relic instrumentation prerequisites.
-2. Play the game, and stop by the **New Relic** booth to show off your game stats dashboard.
-3. Get the high score, and you win!
+2. Play the game, and stop by the **New Relic** booth to show off your game stats dashboard. (you will be rewarded for this effort)
+3. If you have the high score, you win the grand prize!
+
+# Dangerous Dave Replica
+*(this is the description you can find over on https://github.com/mwolfart/dangerous-dave) We remain deeply endebted to them for their effort to bring this classic game to life on the python platform! - Rachel and Leon)*
```

Comment:
To be consistent with the original description, please make the following two paragraphs bullets. I took out the bullets when I created this version.